package.json

```diff
@@ -24,6 +24,7 @@
     "babel-cli": "^6.26.0",
     "babel-plugin-transform-es2015-modules-umd": "^6.24.1",
     "babel-preset-env": "^1.7.0",
+    "jasmine": "^3.1.0",
```

Comment:
Using full jasmine to make it work under nodejs. Maybe we can use just jasmine core in this case too somehow.